src/EducationService.Business/Commands/EducationType/CreateEducationTypeCommand.cs

```diff
@@ -0,0 +1,70 @@
+﻿using LT.DigitalOffice.EducationService.Business.Commands.EducationType.Interfaces;
+using LT.DigitalOffice.EducationService.Data.Interfaces;
+using LT.DigitalOffice.EducationService.Mappers.Db.Interfaces;
+using LT.DigitalOffice.EducationService.Models.Dto.Requests.Education;
+using LT.DigitalOffice.EducationService.Validation.Education.Interfaces;
+using LT.DigitalOffice.Kernel.BrokerSupport.AccessValidatorEngine.Interfaces;
+using LT.DigitalOffice.Kernel.Constants;
+using LT.DigitalOffice.Kernel.FluentValidationExtensions;
+using LT.DigitalOffice.Kernel.Helpers.Interfaces;
+using LT.DigitalOffice.Kernel.Responses;
+using Microsoft.AspNetCore.Http;
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.EducationService.Business.Commands.EducationType
+{
+  public class CreateEducationTypeCommand : ICreateEducationTypeCommand
+  {
+    private readonly IHttpContextAccessor _httpContextAccessor;
+    private readonly IDbEducationTypeMapper _mapper;
+    private readonly IEducationTypeRepository _educationTypeRepository;
+    private readonly ICreateEducationTypeRequestValidator _validator;
+    private readonly IAccessValidator _accessValidator;
+    private readonly IResponseCreator _responseCreator;
+
+    public CreateEducationTypeCommand(
+      IHttpContextAccessor httpContextAccessor,
+      IDbEducationTypeMapper mapper,
+      IEducationTypeRepository educationTypeRepository,
+      ICreateEducationTypeRequestValidator validator,
+      IAccessValidator accessValidator,
+      IResponseCreator responseCreator)
+    {
+      _httpContextAccessor = httpContextAccessor;
+      _mapper = mapper;
+      _educationTypeRepository = educationTypeRepository;
+      _validator = validator;
+      _accessValidator = accessValidator;
+      _responseCreator = responseCreator;
+    }
+
+    public async Task<OperationResultResponse<Guid?>> ExecuteAsync(CreateEducationTypeRequest request)
+    {
+      OperationResultResponse<Guid?> response = new();
```

Comment:
it will be better if you create the variable before 57 line. You do not use it here